scripts/start-encrypted-cluster.sh

```diff
@@ -0,0 +1,27 @@
+
+export CWD=$(pwd);
+mkdir encrypted-cluster
+cd encrypted-cluster
+
+if [ ! -d "drivers-evergreen-tools/" ]; then
+ git clone --depth=1 "https://github.com/mongodb-labs/drivers-evergreen-tools.git"
+fi
+
+export DRIVERS_TOOLS=$CWD/encrypted-cluster/drivers-evergreen-tools
+export MONGODB_VERSION=8.0
+export AUTH=true
+export MONGODB_BINARIES=$DRIVERS_TOOLS/mongodb/bin
+export NODE_DRIVER=~/dev/node-mongodb-native
```

Comment:
```suggestion
```

Reply:
It's not used in this script or in the script that I use locally.  Even if it were, this path wouldn't be accurate unless we also had a copy of the driver cloned at `~/dev/node-mongodb-native` wherever this script is running.